airtime_mvc/public/widgets/widget_schedule_v3.html

```diff
@@ -0,0 +1,21 @@
+<link rel="stylesheet" href="/airtime-widgets/css/airtime-widgets.css" type="text/css" />
+<link rel="stylesheet" href="/airtime-widgets/css/custom.css" type="text/css" />
+<script src="/airtime-widgets/js/jquery-1.6.1.min.js" type="text/javascript"></script>
+<script src="/airtime-widgets/js/jquery-ui-1.8.10.custom.min.js" type="text/javascript"></script>
+<script src="/airtime-widgets/js/jquery.showinfo.js" type="text/javascript"></script>
+<script>
+    $(document).ready(function() {
+        $("#scheduleTabs").airtimeWeekSchedule({
+            sourceDomain:"https://airtime.97irratia.info/",
```

Comment:
I think we'll need to make this URI dynamic.

Reply:
Ups, yeah.. nice catch